c2corg_ui/static/js/documentediting.js

```diff
@@ -551,10 +589,23 @@ app.DocumentEditingController.prototype.formatOuting_ = function(outing) {
     // init empty conditions_levels for ng-repeat
     outing['locales'][0]['conditions_levels'] = [{'level_snow_height_soft': '', 'level_snow_height_total': '', 'level_comment': '', 'level_place': ''}];
   }
-
   return outing;
 }
 
+/**
+ * @returns {Object} url parameters
+ * @private
+ */
+app.DocumentEditingController.prototype.parseUrl_ = function() {
+  var query = decodeURIComponent(location.search.substr(1));
+  var result = {};
+  query.split('?').forEach(function(part) {
```

Comment:
FYI: This is how the function, that is used by $location, is implemented in Angular:
https://github.com/angular/angular.js/blob/5d695e5566212d93da0fc1281d5d39ffee0039a3/src/Angular.js#L1321-L1350

But we can keep it simple and use your function, that works in our case.